README.md

```diff
@@ -146,5 +152,5 @@ before the application has finished it's cleanup, a 'SIGKILL' signal is send
 to the process (which results in an immediate termination, and this signal
 can't be caught).
 
-[badgepub]: https://circleci.com/gh/scality/Arsenal.svg?style=svg
-[badgepriv]: http://ci.ironmann.io/gh/scality/Arsenal.svg?style=svg&circle-token=c3d2570682cba6763a97ea0bc87521941413d75c
+[badgepub]: https://github.com/scality/Arsenal/actions/workflows/tests.yaml/badge.svg
+[codecov]: https://codecov.io/gh/scality/Arsenal/branch/development/8.1/graph/badge.svg?token=X0esXhJSwb
```

Comment:
is this token "safe" to share here: where does it come from/what does it give access to?

Just tried without the token, and it seems to work jsut fine: even in a new "private browsing" window, so i guess we can just remove it...

```suggestion
[codecov]: https://codecov.io/gh/scality/Arsenal/branch/development/8.1/graph/badge.svg
```